target/linux/ath79/dts/qca9558_tplink_tl-wr1043nd.dtsi

```diff
@@ -124,19 +124,102 @@
 &mdio0 {
```

Comment:
I had to change the names of the network ports and the led configuration to make this pull request work on tl-wr1043nd-v2

```
diff --git a/target/linux/ath79/dts/qca9558_tplink_tl-wr1043nd.dtsi b/target/linux/ath79/dts/qca9558_tplink_tl-wr1043nd.dtsi
index b4d956f04f..d9a7227018 100644
--- a/target/linux/ath79/dts/qca9558_tplink_tl-wr1043nd.dtsi
+++ b/target/linux/ath79/dts/qca9558_tplink_tl-wr1043nd.dtsi
@@ -130,6 +130,8 @@
                #size-cells = <0>;
 
                reg = <0>;
+               qca,ignore-power-on-sel;
+               qca,led-open-drain;
 
                ports {
                        #address-cells = <1>;
@@ -150,35 +152,35 @@
 
                        port@1 {
                                reg = <1>;
-                               label = "wan";
+                               label = "lan4";
                                phy-mode = "internal";
                                phy-handle = <&phy_port1>;
                        };
 
                        port@2 {
                                reg = <2>;
-                               label = "lan4";
+                               label = "lan3";
                                phy-mode = "internal";
                                phy-handle = <&phy_port2>;
                        };
 
                        port@3 {
                                reg = <3>;
-                               label = "lan3";
+                               label = "lan2";
                                phy-mode = "internal";
                                phy-handle = <&phy_port3>;
                        };
 
                        port@4 {
                                reg = <4>;
-                               label = "lan2";
+                               label = "lan1";
                                phy-mode = "internal";
                                phy-handle = <&phy_port4>;
                        };
 
                        port@5 {
                                reg = <5>;
-                               label = "lan1";
+                               label = "wan";
                                phy-mode = "internal";
                                phy-handle = <&phy_port5>;
                        };
```

Reply:
I checked the config. This is correct. Updated.